### ISSUE TITLE
Extend middleware to http4s client

### ISSUE THREAD

**Issue by PashoQ** [novakov-alexey/http4s-spnego#42]
Is it possible to extended this middleware to  htt4s client?
Like requesting token from GSSManager and add "Authorization: Negotiate $token" header to client request.

**Comment by novakov-alexey** [comment]
Do you mean smth like this? https://medium.com/se-notes-by-alexey-novakov/spnego-token-generation-90c4267f784e
It is in example how to get a token using Java and then just add a token to the Authorization header.

**Comment by PashoQ** [comment]
Yes. It would be nice that it would be available from library itself. I use server middleware from this library as well.

**Comment by novakov-alexey** [comment]
@PashoQ I think this needs to be done in the http4s blaze client to make such token added to the request
